samples/ra-vpn/saml-multi-region/ca-certificate.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
```

Comment:
would be good to have a one line comment on what this cert is used for..

Reply:
added